pontoon/base/static/js/translate.js

```diff
@@ -320,18 +320,18 @@ var Pontoon = (function (my) {
                 '" title="Copy Into Translation (Tab)">' +
                   '<header class="clearfix' +
                     ((self.user.isTranslator) ? ' translator' :
-                      ((self.user.email === this.email && !this.approved) ?
+                      ((self.user.uid === this.uid && !this.approved) ?
                         ' own' : '')) +
                     '">' +
                     '<div class="info">' +
-                      ((!this.email) ? '<span title="' + self.getApproveButtonTitle(this) + '">' + this.user + '</span>' :
+                      ((!this.uid) ? '<span>' + this.user + '</span>' :
```

Comment:
It hasn't any useful info because this condition will work only if translation is "imported". In this state we don't know who appoved or did anything else with this string.